src/resources/css/tribe-events-full.css

```diff
@@ -1904,3 +1914,126 @@ body.tribe-theme-twentysixteen .datepicker table {
 		background-image: url(../images/events-bar/icon-map@2x.png);
 	}
 }
+
+/* = List View Template Styles
+   @ Template: condensed-list.php
+=============================================*/
+
+.tribe-events-condensed-list .tribe-events-page-title {
+    margin: 0 0 1em 0;
+}
+
+.tribe-events-condensed-list .type-tribe_events {
+    border-bottom: 2px solid #eaeaea;
+}
+
+@media (max-width: 414px) {
+    .tribe-events-condensed-list .type-tribe_events {
+        padding-bottom: 1em;
+        margin-bottom: 1em;
+    }
+}
+
+table.tribe-events-calendar-condensed-list {
+    border-collapse: collapse;
+    clear: both;
+    font-size: 80%;
+    table-layout: fixed;
+    width: 100%;
+    border: none;
+}
+
+@media (max-width: 414px) {
```

Comment:
I would suggest wrapping all styles for `max-width: 414px` in the same media query.